content/en/api/downtimes/code_snippets/api-monitor-schedule-downtime.rb

```diff
@@ -11,16 +11,30 @@
 end_reccurrence_ts = start_ts + (4 * 7 * 24 * 60 * 60) # 4 weeks from now
 
 recurrence = {
-  'type' => 'weeks',
-  'period' => 1,
-  'week_days' => %w[Mon Tue Wed Thu Fri],
-  'until_date' => end_reccurrence_ts
+    'type' => 'weeks',
+    'period' => 1,
+    'week_days' => %w[Mon Tue Wed Thu Fri],
+    'until_date' => end_reccurrence_ts
 }
 
 # Schedule downtime
-dog.schedule_downtime(
-  'env:testing',
-  start: start_ts,
-  end: end_ts,
-  recurrence: recurrence
+api.Downtime.create(
```

Comment:
Looks like this line is for the python client? I think this needs to be the `dog.schedule_downtime` as it was before?

Reply:
you're correct, it was that before. will update.